### ISSUE TITLE
Falrin Treeshaper doesn't provide Banner of Provocation for second, third.... time.

### ISSUE THREAD

**Issue by Leinhart1111** [slowtorta/turtlewow-bug-tracker#5828]
**Please, provide a clear description what the bug is:**

**Steps to reproduce the behavior with as much detail as possible:**

1. Falrin Treeshaper just provides text when you talk to him.
2. 
3. 

**IDs of affected creatures, items, quests or spells with a link to the relevant page:**

NPC: https://database.turtle-wow.org/?npc=16032
Item: https://database.turtle-wow.org/?item=21986
Quest: https://database.turtle-wow.org/?quest=
Spell: https://database.turtle-wow.org/?spell=
Object: https://database.turtle-wow.org/?object=

**Expected behavior. Describe how it should work:**
Falrin Treeshaper should provide a Banner of Provocation when talked to after completed the mission of the The Challenge

I destroyed the Banner and I want to summon Theldren again for the amulet for the T 0.5

**Comment by twowTurtReynolds** [comment]
I do not understand. Have you completed https://database.turtle-wow.org/?quest=9015 yet or not? 

If you have, you keep the Provcation banner item. If you delete it, there is no way to get it back, blizzlike. If you can prove otherwise, please let us know.

**Comment by Leinhart1111** [comment]
Yes, I have. And look people says that it can be restored both the amulet and the Banner but here none can be restored and there is no option to get them again.

https://www.wowhead.com/classic/item=22047/top-piece-of-lord-valthalaks-amulet#comments
https://www.wowhead.com/classic/item=21986/banner-of-provocation#comments

I dont know what happened to my amulet so I tried to get the Banner back but there's no option, the Banner i tossed it.

**Comment by twowTurtReynolds** [comment]
Which quest are you currently on? Can you take a picture of Bodley's dialogue?

**Comment by Leinhart1111** [comment]
Im on this quest:
https://www.wowhead.com/classic/quest=8989/the-right-piece-of-lord-valthalaks-amulet

And Bodley's dialogues are: 
![Captura de pantalla (297)](https://github.com/slowtorta/turtlewow-bug-tracker/assets/155205107/8de97581-8ffc-4d25-8ced-fecfeace230b)
![Captura de pantalla (300)](https://github.com/slowtorta/turtlewow-bug-tracker/assets/155205107/9d8ceba7-3083-4830-8e3f-88903eeada19)
![Captura de pantalla (301)](https://github.com/slowtorta/turtlewow-bug-tracker/assets/155205107/209d24a5-ece5-4848-acab-dae21197e793)
![Captura de pantalla (302)](https://github.com/slowtorta/turtlewow-bug-tracker/assets/155205107/cf27c9d3-a294-4d3e-97be-e98767ce86d5)
![Captura de pantalla (303)](https://github.com/slowtorta/turtlewow-bug-tracker/assets/155205107/ae7a5bac-6e76-424e-8cef-70016dfb5f51)
![Captura de pantalla (304)](https://github.com/slowtorta/turtlewow-bug-tracker/assets/155205107/b6824b70-575a-434c-897e-70e9fea8a964)
![Captura de pantalla (305)](https://github.com/slowtorta/turtlewow-bug-tracker/assets/155205107/cedd695d-efa5-4630-86ed-518fc47f3a90)
![Captura de pantalla (299)](https://github.com/slowtorta/turtlewow-bug-tracker/assets/155205107/e4ceee92-a84d-4f5b-9cb9-03b9c2c04213)

**Comment by twowTurtReynolds** [comment]
There are 4 variants of the Right Piece quest:
https://database.turtle-wow.org/?search=the%20right%20piece

Please select the one you did.

**Comment by Leinhart1111** [comment]
I did the Stratholme quest, summoned the twins and got the piece and you can see it in my bags but I'm still missing the Amulet and I can't get it because I can't summon Theldren anymore

**Comment by Leinhart1111** [comment]
https://database.turtle-wow.org/?quest=8991

**Comment by twowTurtReynolds** [comment]
Sorry this is a long quest for me to run through, I'm getting around to it now. I think you are right about this lacking missing feature, but I'm doing the quest in full to fully verify that there isnt a misunderstanding here.

**Comment by twowTurtReynolds** [comment]
@Leinhart1111 did you figure it out, was there a misunderstanding? Or did a GM just spawn the item for you? If the issue is actually not fixed, then we still need to fix it for others.
